Formula/gor.rb

```diff
@@ -2,10 +2,10 @@
 
 class Gor < Formula
   desc "Real-time HTTP traffic replay tool written in Go"
-  homepage "https://github.com/buger/gor/"
-  url "https://github.com/buger/gor/archive/v0.10.1.tar.gz"
-  sha256 "283ca037a782844df42a0352c072efb558ffca3dc76f88a6317eca0d44ab1a5c"
-  head "https://github.com/buger/gor.git"
+  homepage "https://gortool.com"
+  url "https://github.com/buger/gor.git",
+      :tag => "v0.13.0",
+      :revision => "98fe7f19b21eaaffcd0d733a57668f63de5921d7"
```

Comment:
Yes, it's significantly slower for users and more expensive for GitHub. Please move it back to the old way, thanks.